plugins/modules/ntc_config_command.py

```diff
@@ -135,115 +139,116 @@
       - end
     provider: "{{ nxos_provider }}"
 
-'''
-
+"""
 import os.path
 import socket
+
+from ansible.module_utils.ansible_release import __version__ as ansible_version
+from ansible.module_utils.basic import AnsibleModule
+
 try:
     from netmiko import ConnectHandler
-    HAS_NETMIKO=True
+
+    HAS_NETMIKO = True
 except ImportError:
-    HAS_NETMIKO=False
+    HAS_NETMIKO = False
 
 
-from ansible import __version__ as ansible_version
 try:
     from packaging import version
-    HAS_PACKAGING=True
+
+    HAS_PACKAGING = True
 except ImportError:
-    HAS_PACKAGING=False
-if (HAS_PACKAGING and version.parse(ansible_version) < version.parse("2.4")) or (not HAS_PACKAGING and float(ansible_version[:3]) < 2.4):
+    HAS_PACKAGING = False
+if (HAS_PACKAGING and version.parse(ansible_version) < version.parse("2.4")) or (
+    not HAS_PACKAGING and float(ansible_version[:3]) < 2.4
+):
```

Comment:
We're pinning our ansible version to 2.10 in our pyproject.toml so we can get rid of this version checking code that `ansible-test sanity` doesn't like